zeppelin-web/src/app/interpreter/interpreter.html

```diff
@@ -286,7 +299,7 @@ <h3 class="interpreter-title">{{setting.name}}
             <td>
               <textarea ng-if="valueform.$visible" ng-model="dep.exclusions"
                         placeholder="(Optional) comma separated groupId:artifactId list"
-                        form="valueform"
+                        e-form="valueform"
```

Comment:
@corneadoug 
I attach a link, part of the note.
https://github.com/apache/zeppelin/pull/1395/files#diff-52843fa71a36ded9162a8bbb812459bcR295

I do not know the correct action on the part of the fact.
However, this part will be described as `Invalid value` in my IDE.
<img width="559" alt="2016-09-06 10 52 13" src="https://cloud.githubusercontent.com/assets/10525473/18259547/13b817aa-7420-11e6-85a2-97191940b2e2.png">
e-form will resolve this issue.